buildscripts/incremental/setup_conda_environment.cmd

```diff
@@ -26,7 +26,7 @@ conda create -n %CONDA_ENV% -q -y python=%PYTHON% numpy=%NUMPY% cffi pip scipy j
 
 call activate %CONDA_ENV%
 @rem Install latest llvmlite build
-%CONDA_INSTALL% -c numba/label/dev llvmlite=0.39
+%CONDA_INSTALL% -c numba/label/dev llvmlite=0.40.0dev0
```

Comment:
```suggestion
%CONDA_INSTALL% -c numba/label/dev llvmlite=0.40
```

This will have the advantage that, by the time we do release a `0.40.0rc1` public CI will pick that up. Using `0.40` as version spec will pickup an `rc1` as evidenced by the following snippet:

```
zsh» conda install -c numba numba=0.56
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: /Users/esc/miniconda3/envs/test_0.56rc1

  added / updated specs:
    - numba=0.56


The following NEW packages will be INSTALLED:

  blas               pkgs/main/osx-64::blas-1.0-mkl
  intel-openmp       pkgs/main/osx-64::intel-openmp-2021.4.0-hecd8cb5_3538
  llvm-openmp        pkgs/main/osx-64::llvm-openmp-12.0.0-h0dcd299_1
  llvmlite           numba/osx-64::llvmlite-0.39.0rc1-py310_0
  mkl                pkgs/main/osx-64::mkl-2021.4.0-hecd8cb5_637
  mkl-service        pkgs/main/osx-64::mkl-service-2.4.0-py310hca72f7f_0
  mkl_fft            pkgs/main/osx-64::mkl_fft-1.3.1-py310hf879493_0
  mkl_random         pkgs/main/osx-64::mkl_random-1.2.2-py310hc081a56_0
  numba              numba/osx-64::numba-0.56.0rc1-np1.21py3.10hf701924_gea84072_0
  numpy              pkgs/main/osx-64::numpy-1.22.3-py310hdcd3fac_0
  numpy-base         pkgs/main/osx-64::numpy-base-1.22.3-py310hfd2de13_0
  six                pkgs/main/noarch::six-1.16.0-pyhd3eb1b0_1


Preparing transaction: done
Verifying transaction: done
Executing transaction: done
  4.09s user 2.47s system 68% cpu 9.537 total
```